minio/overlays/nerc-ocp-obs/files/minio-config.env

```diff
@@ -0,0 +1,11 @@
+# Documentation: https://min.io/docs/minio/linux/reference/minio-server/settings/iam/openid.html
+
+MINIO_IDENTITY_OPENID_CONFIG_URL=https://dex-dex.apps.ocp-obs.nerc.mghpcc.org/.well-known/openid-configuration
```

Comment:
One more thing @dheerajodha , we need to change `ocp-obs` to `obs` here.

Reply:
Thanks! updated.